README.rst

```diff
@@ -54,16 +54,51 @@ Client(host, port, auto_reconnect)
 
     auto_reconnect - Optional argument to enable/disable automatic reconnect when connection is lost. Defaults to True if not specified.
 
+    user_id - Optional argument to use when server requires authentication - user id (username)
+
+    password  - Optional argument to use when server requires authentication - user password
+
+    use_encryption - Optional boolean argument to set when server uses encryption
+
+    encryption_parameters - Optional argument to set encryption parameters, like TLS certificates verification. Parameter is compatible with python websocket client 'sslopt' parameter. For more information see https://pypi.org/project/websocket_client
+
 - Returns
 
     The connected client object.
 
-- Usage
+- Usage example
 
     .. code:: python
 
         client = cdp.Client('127.0.0.1')
 
+- Usage example with password authentication
+
+    .. code:: python
+
+        client = cdp.Client('127.0.0.1', user_id="test", password="12345678")
+
+- Usage example with password authentication and encryption in use, without server certificate verification
+
+    .. code:: python
+
+        import ssl
+
+        client = cdp.Client('127.0.0.1', user_id="test", password="12345678", use_encryption=True,
```

Comment:
for consistency first parameter should also be 
host="127.0.0.1" if possible

Reply:
Done